### DISCUSSION TITLE
[FR] Support using the first item to fold following items when no child items exist.

### DISCUSSION THREAD

**Discussion by xmbai9507** [vslinko/obsidian-outliner#434] [Archive]
Kind of like what an IDE does to the comments.
![image](https://user-images.githubusercontent.com/65625632/235393212-7bc51cc6-6382-41bf-89a8-468533acaede.png)

Not sure if it is feasible in Obsidian, but if a list has no child item, hope to have a folder marker attached to the first item to toggle folding the following items of the same indentation level.

**Comment by vslinko** [comment]
@XMB-7 Unfortunately, I can't do that in the plugin. It's built-in editor behavior. Please suggest your idea directly to Obsidian.
